### DISCUSSION TITLE
Region everywhere?

### DISCUSSION THREAD

**Discussion by ziaan2** [Pryaxis/TShock#2907] [Q&A] [answered]
i was making a terraria survival ssc server for a discord server , i tried testing with alt character but there is a region thats blocking mine/place everywhere and i cant do anything other then attack enemies
i am using a hosting website and ssc is enabled

**Comment by ziaan2** [comment]
i also forgot to say that i did not put the region in it and i havent share the server to anyone

**Comment by sgkoishi** [comment, marked as answer]
Check if you are building inside spawn, or have `/antibuild` enabled.

**Reply by ziaan2** [reply]
i build outside spawn and i disabled antibuild

**Reply by sgkoishi** [reply]
What error message did you see? I would guess you are missing the basic `tshock.world.modify` permission.

**Reply by ziaan2** [reply]
just a "You do not have permission to build in this region"

**Reply by sgkoishi** [reply]
Run `/userinfo <your name>` and `/group listperm <your group>` to see if you have the basic `tshock.world.modify` permission.

**Reply by ziaan2** [reply]
thanks it worked
